Docs/source/developers/profiling.rst

```diff
@@ -151,6 +191,13 @@ Example on how to create traces on a multi-GPU system that uses the Slurm schedu
    This version does not record all trace information.
    You need to use the one directly shipped with the NVHPC base system, version 2021.4.1, located in ``/opt/nvidia/hpc_sdk/Linux_x86_64/21.11/compilers/bin/nsys``.
 
+.. note::
+
+    If everything went well, the user will obtain as many output files named "profiling_<number>.nsys-rep" as active GPU's which will allow the user to       analyze each of those GPU's perforamces with Nsight system.
```

Comment:
Great idea to add what is expected!

Here are small suggestions: make the file name `verbatim` and remove some spaces.
```suggestion
    If everything went well, the user will obtain as many output files named ``profiling_<number>.nsys-rep`` as active GPU's, which will allow the user to analyze each GPU's performance with Nsight system.
```